### ISSUE TITLE
information about the pretrained ResNet-preact-8 model 

### ISSUE THREAD

**Issue by ShreshthSaxena** [hysts/pytorch_mpiigaze#44]
Hi,
Can you please share details about the ResNet-preact-8 pre-trained model that's used in this repository ? What's the inspiration behind this and how much does this differ from the author's LeNet approach ?

**Comment by hysts** [comment]
Hi, @ShreshthSaxena 

Basically, I just wondered what if I used a stronger backbone, so I tried it out in this repo. There's not much difference, it’s just that a bit slower than LeNet, but more accurate and reasonably fast.

In my understanding, at the time the original paper was written by the authors, the ResNet paper had not been published yet, and the authors only tried simple LeNet.

As I wanted a fast and accurate model, I tried some shallow ResNets, but it didn't improve much on the deeper models, so I chose ResNet-8 for the pre-training model provided in this repo.
